src/applications/SimulationResults.java

```diff
@@ -72,22 +72,24 @@ public void setJobCompletionData(int jobNumber, int completionTime, int totalWai
 	 * */
 	void outputStatistics(MachineShopSimulator machineShopSimulator) {
 	    setFinishTime(machineShopSimulator.getTimeNow());
-	    setNumMachines(machineShopSimulator.numMachines);
+	    setNumMachines(machineShopSimulator.getNumMachines());
 	    setNumTasksPerMachine(machineShopSimulator);
 	    setTotalWaitTimePerMachine(machineShopSimulator);
 	}
 
 	void setTotalWaitTimePerMachine(MachineShopSimulator machineShopSimulator) {
-	    int[] totalWaitTimePerMachine = new int[machineShopSimulator.numMachines+1];
-	    for (int i=1; i<=machineShopSimulator.numMachines; ++i) {
+        int numOfMachines = machineShopSimulator.getNumMachines();
+	    int[] totalWaitTimePerMachine = new int[numOfMachines+1];
+	    for (int i=1; i<=numOfMachines; ++i) {
```

Comment:
should be i++

Reply:
I think here there is no difference in whether we have i++ or ++i. I am pretty sure it only maters when we are using the return value of the operation, but here we are only updating i.
I think just whichever is clearer is better.
[This](https://stackoverflow.com/questions/24853/c-what-is-the-difference-between-i-and-i) explains the difference, but does so regarding C (I'm pretty sure it's the same in Java).

Reply:
I am in favor of i++ mostly since is seems to be standard, There is a difference between the two, it just not relevant here, but I usually have felt i++ is more clearer